web/samples/uncompiled.html

```diff
@@ -46,6 +46,7 @@
     <script>
       var kmw=window.tavultesoft.keymanweb;
       kmw.init({
+		attachType:'auto',
```

Comment:
Is `attachType` now required? Or did you do this for testing against touch platforms?